keras_nlp/models/xlm_roberta/xlm_roberta_tokenizer.py

```diff
@@ -100,10 +100,11 @@ def __init__(self, proto, **kwargs):
         self.pad_token_id = 1  # <pad>
         self.end_token_id = 2  # </s>
         self.unk_token_id = 3  # <unk>
+        self.mask_token_id = self.vocabulary_size() - 1  # <mask>
```

Comment:
I did not carefully read XLM-R tokenization part, will let Matt check this. @mattdangerw